artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ha/ReplicaPolicy.java

```diff
@@ -57,6 +57,8 @@
 
    private long voteRetryWait;
 
+   private int quorumVoteWait = ActiveMQDefaultConfiguration.getDefaultQuorumVoteWait();
```

Comment:
It would be better to just use the one provided by the constructor and declare this `final` if is not meant to be modified anymore